doc/classes/World3D.xml

```diff
@@ -13,6 +13,9 @@
 		<member name="camera_attributes" type="CameraAttributes" setter="set_camera_attributes" getter="get_camera_attributes">
 			The default [CameraAttributes] resource to use if none set on the [Camera3D].
 		</member>
+		<member name="compositor" type="Compositor" setter="set_compositor" getter="get_compositor">
+			The World3D's [Compositor]. Set by [WorldEnvironment] nodes in this World3D.
```

Comment:
Should mention `Camera3D` IMO as it will be overridden by any `Camera3D` as well

I also think the meaning of "this `World3D`" is ambiguous

Reply:
can you tell me how it's ambiguous or suggest an alternative wording? because i've seen plenty different ways (such as "registered to the World3D") to describe "being a child of a viewport that has a world" in the docs, and just can't find a neat phrasing. there's also the concept of "entering/exiting" a "world" (see the notifications on `Node3D` and the *undocumented* virtuals `Node3D._{enter,exit}_world()->void`), so i figured "being in it" makes sense?

Reply:
Does the wording "in this World3D" occur anywhere else? Because to me it isn't necessarily clear what that means, so any other wordings would be useful as a guide for how to write it. Also I think the "set by" should be "will be overridden by" to clarify that this isn't just some property for the engine to use and that users are free to use it too. I'd suggest:
```suggestion
			The default [Compositor] resource to be use if none is set on the [Camera3D]. Will be replaced by the first [WorldEnvironment] registered to this World3D.
```
Using the wording from `Node3D` with `registered`.

Reply:
> Does the wording "in this World3D" occur anywhere else?

not really, it just made most sense to me.

> overridden by

sounds to me like it technically stays its value, but not be in effect, but that might just be some weird connotation from some weird standard i read where it meant that.

> I'd suggest:... Using the wording from Node3D with registered.

I'll take it, thanks, i'm just not super happy about the Node3D wording to begin with, because *technically* the node knows which world it belongs to, not the world which nodes belong to it, because the world is just a resource property of the viewport.

but honestly if someone's that deep into the implementation detail they'll know what it means :)